charts/nirmata/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: kyverno
-version: 3.1.0-rc2
-appVersion: v1.11.0-n4k.nirmata.2
+version: 3.1.0
+appVersion: v1.11.4-n4k.nirmata.1
```

Comment:
No, we cannot put 3.1.0. Else, that will become the default chart version instelled. Need to keep it at rc1,2,3, etc. Till 1.11 becomes the default.

Reply:
Got it